src/lib/viewers/controls/rotate/RotateControl.scss

```diff
@@ -0,0 +1,5 @@
+@import '../styles';
+
+.bp-RotateControl {
+    @include bp-ControlButton();
```

Comment:
We can leave the `()` off the end of a standard import.

Reply:
Got it, will change it!